guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -129,7 +129,14 @@ ifdef::satellite[]
 ----
 endif::[]
 endif::[]
+
 +
+[NOTE]
+====
+With the enablement of the module `satellite:el8` warns about a conflict with `postgresql:10` and `ruby:2.5` as these modules are set to the default module versions on {RHEL} 8. The module `satellite:el8` has a dependency for the modules `postgresql:12` and `ruby:2.7` that will be enabled with the `satellite:el8` module. These warnings do not cause installation process failure, hence can be ignored safely.
```

Comment:
This should be behind an `ifdef`. It should only show up in the satellite manual. In `foreman-el` the module is named `foreman` and in `katello` it's named `katello` so the instructions would be incorrect. `foreman-deb` has no modules at all. Not sure about `orcharhino`, but for now I'd hide it behind `ifdef::satellite[]`.

Also, I'm not sure about the exact phrasing. Can't it be simplified to:
```suggestion
Enablement of the module `satellite:el8` warns about a conflict with `postgresql:10` and `ruby:2.5` as these modules are set to the default module versions on {RHEL} 8. The module `satellite:el8` has a dependency for the modules `postgresql:12` and `ruby:2.7` that will be enabled with the `satellite:el8` module. These warnings do not cause installation process failure, hence can be ignored safely.
```

Reply:
Yeah, just move it inside the `ifdef::satellite` on line 128 perhaps?